src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -37,15 +34,12 @@ const useGroupInfo = () => {
   const createChat = channelState.channel?.data?.created_at;
   const countUser = Object.entries(participants).length;
   const [selectedUser, setSelectedUser] = React.useState(null);
-  const [newParticipant, setNewParticipan] = React.useState([]);
+  const [newParticipant, setNewParticipant] = React.useState([]);
   const [openModal, setOpenModal] = React.useState(false);
   const [isAnonymousModalOpen, setIsAnonymousModalOpen] = React.useState(false);
   const [isFetchingAllowAnonDM, setIsFetchingAllowAnonDM] = React.useState(false);
   const [, dispatchChannel] = React.useContext(Context).channel;
-
-  const {goToChatScreen} = useChatUtilsHook();
-  const {localDb} = useLocalDatabaseHook();
-
+  const {createSignChat, handleAnonymousMessage} = useCreateChat();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable name `newParticipant` has been corrected from `newParticipan`. This is a good change as it improves readability and avoids potential confusion in the future.

```diff
- const [newParticipant, setNewParticipan] = React.useState([]);
+ const [newParticipant, setNewParticipant] = React.useState([]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->